src/Features/LanguageServer/Protocol/Features/Options/AutoFormattingOptionsStorage.cs

```diff
@@ -17,15 +18,21 @@ public static AutoFormattingOptions GetAutoFormattingOptions(this IGlobalOptionS
             FormatOnCloseBrace = globalOptions.GetOption(FormatOnCloseBrace, language)
         };
 
+    private static readonly OptionGroup s_formattingGroup = new(
```

Comment:
I feel we might want to clean & add option group for all the non-editorconfig options.
It's another pretty big change, if this looks good I can create a following up work item.